components/brave_ads/browser/ads_status_header_throttle_unittest.cc

```diff
@@ -17,16 +18,21 @@ namespace {
 
 constexpr char kAdsStatusHeader[] = "X-Brave-Ads-Enabled";
 constexpr char kAdsEnabledStatusValue[] = "1";
-constexpr char kAllowedURL[] = "https://search.brave.com/search";
-constexpr char kNotAllowedURL[] = "https://brave.com/search";
+constexpr const char* kAllowedHosts[] = {
+    "search.brave.com",          "search.brave.software",
+    "search.bravesoftware.com",  "safesearch.brave.com",
+    "safesearch.brave.software", "safesearch.bravesoftware.com",
+    "search-dev-local.brave.com"};
+constexpr char kNotAllowedHost[] = "brave.com";
 constexpr char kTestingHeaderName[] = "TestingHeaderName";
 constexpr char kTestingHeaderValue[] = "TestingHeaderValue";
 
-network::ResourceRequest BuildRequest() {
+network::ResourceRequest BuildRequest(const char* host) {
   network::ResourceRequest request;
-  request.url = GURL(kAllowedURL);
+  const std::string url = base::StrCat({"https://", host, "/search"});
```

Comment:
constexpr for `https://` and `/search`

Reply:
nit: a URL as a string is known as a spec, so maybe `spec =`

Reply:
fixed